src/components/ListboxContainer.tsx

```diff
@@ -19,37 +19,32 @@ const ListboxContainer = ({
   const [listboxInstance, setListboxInstance] = useState<stardust.FieldInstance>();
   const elRef = useRef<HTMLElement>();
 
+  const { embed, model, selectionsApi } = store.getState();
+
   useEffect(() => {
-    if (!app) {
+    if (!layout || !embed) {
```

Comment:
will it ever happen that embed doesn't exist you think?

Reply:
Otherwise TS is complaining about it. It's because we initialise embed as `undefined` in the store.